x-pack/plugins/security_solution/server/lib/detection_engine/routes/index/ecs_mapping.json

```diff
@@ -1,12 +1,37 @@
 {
+  "index_patterns": [
+    "try-ecs-*"
+  ],
   "mappings": {
-    "dynamic": false,
+    "_meta": {
```

Comment:
I opted to leave these top-level fields because:

1. we're not using anything but `.mappings.properties`
2. it makes ECS updates easier as it's just a copy/paste from the [generated mappings template.json](https://github.com/elastic/ecs/blob/1.9/generated/elasticsearch/7/template.json)

Reply:
and if we did accidentally start using them, the snapshot test would tell us so 😉